docs/src/content/docs/pt/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Starlight 🌟 Construa sites de documentação com Astro
+head:
+  - tag: title
+    content: Starlight 🌟 Construa sites de documentação com Astro
+description: Starlight ajuda-te a construir websites de documentação belos e rápidos com Astro.
+template: splash
+editUrl: false
+lastUpdated: false
+hero:
+  title: Faça a sua documentação brilhar com Starlight
+  tagline: Tudo o que precisa para construir um site de documentação brilhante. Rápido, acessível e fácil de usar.
+  image:
+    file: ~/assets/hero-star.webp
+  actions:
+    - text: Começar
+      icon: right-arrow
+      variant: primary
+      link: /pt-br/getting-started/
+    - text: Veja no GitHub
+      icon: external
+      link: https://github.com/withastro/starlight
+---
+
+import { CardGrid, Card } from '@astrojs/starlight/components';
+import AboutAstro from '~/components/about-astro.astro';
+
+<CardGrid stagger>
+	<Card title="Documentação de iluminar os olhos" icon="open-book">
+		Inclui: Navegação do site, pesquisa, internacionalização, SEO, tipografia
+		fácil de ler, <i>highlighting</i> de código, modo escuro e mais.
+	</Card>
+	<Card title="Possibilitado pelo Astro" icon="rocket">
+		Aproveite todo o poder e performance do Astro. Estenda o Starlight com suas
+		integrações e bibliotecas favoritas do Astro.
+	</Card>
+	<Card title="Markdown, Markdoc e MDX" icon="document">
+		Utilize a sua linguagem de marcação favorita. O Starlight inclui validação com
+		<i>frontmatter</i> e segurança integrada de tipos com TypeScript.
+	</Card>
+	<Card title="Traga seus componentes de UI" icon="puzzle">
+		O Starlight é uma solução completa de documentação agnóstica a frameworks.
+		Estenda-o com React, Vue, Svelte, Solid e outros.
+	</Card>
+</CardGrid>
+
+<AboutAstro title="Trazido por">
```

Comment:
Seems like this section got outdated, there is a whole new testimonials section before this now

Reply:
Added!